applications/RANSApplication/custom_elements/data_containers/k_epsilon/epsilon_element_data.cpp

```diff
@@ -86,61 +104,25 @@ void EpsilonElementData<TDim>::CalculateGaussPointData(
     this->GetConstitutiveLaw().CalculateValue(cl_parameters, EFFECTIVE_VISCOSITY, mKinematicViscosity);
     mKinematicViscosity /= mDensity;
 
-    double tke;
-
     FluidCalculationUtilities::EvaluateInPoint(
         this->GetGeometry(), rShapeFunctions, Step,
-        std::tie(tke, TURBULENT_KINETIC_ENERGY),
+        std::tie(mTurbulentKineticEnergy, TURBULENT_KINETIC_ENERGY),
         std::tie(mTurbulentKinematicViscosity, TURBULENT_VISCOSITY),
         std::tie(mEffectiveVelocity, VELOCITY));
 
-    mGamma = KEpsilonElementData::CalculateGamma(mCmu, tke, mTurbulentKinematicViscosity);
+    mGamma = KEpsilonElementData::CalculateGamma(mCmu, mTurbulentKineticEnergy, mTurbulentKinematicViscosity);
 
-    mVelocityDivergence =
-        GetDivergence(this->GetGeometry(), VELOCITY, rShapeFunctionDerivatives);
+    FluidCalculationUtilities::EvaluateGradientInPoint(
+        this->GetGeometry(), rShapeFunctionDerivatives,
+        std::tie(mVelocityGradient, VELOCITY));
```

Comment:
this use of `tuple`s is a bit too esoteric for my taste. The problem it solves (identifying which variable a value belongs to) seems to come up pretty often, so we cook up a more readable and intuitive solution

... but I guess that's way beyond the scope of this PR.